qa/tasks/deepsea.py

```diff
@@ -1908,6 +1908,28 @@ def wait_for_health_ok(self, **kwargs):
                                "after reaching timeout"
                                .format(cluster_status))
 
+    def drive_group_modify_limit_by(self, **kwargs):
+        """
+        Helper to modify drive_groups limit field value
+
+        ex.
+        drive_group_modify_limit_by:
+            limit: -1
+
+        this will take all number of disks, reduce it by OS reserved
+        and then reduce/add by the limit value
+
+        Used to cover scenarios for drive-replacement test on OpenStack and HW
+        """
+        number_of_disks = self.master_remote.sh("sudo lsblk -d -n | wc -l")
+        os_reserved_disk = 1
```

Comment:
this is a boot disk

Reply:
Dropped. WIll be handled by `ceph-volume inventory` output